.github/workflows/build-addon-on-push.yml

```diff
@@ -86,12 +86,21 @@ jobs:
       - name: Install scons
         run: |
           python -m pip install scons==4.0.0
-      - name: Create extension library
+      - name: Create Desktop extension library
         run: |
           cd aar
           scons platform=${{ matrix.platform }} target=template_debug ${{ matrix.flags }} custom_api_file=thirdparty/godot_cpp_gdextension_api/extension_api.json build_profile=thirdparty/godot_cpp_build_profile/build_profile.json
           scons platform=${{ matrix.platform }} target=template_release ${{ matrix.flags }} custom_api_file=thirdparty/godot_cpp_gdextension_api/extension_api.json build_profile=thirdparty/godot_cpp_build_profile/build_profile.json
           cd ..
+      - name: Create Android extension library
+        run: |
+          cd aar
+          scons platform=${{ matrix.platform }} target=template_debug arch=arm64 custom_api_file=thirdparty/godot_cpp_gdextension_api/extension_api.json build_profile=thirdparty/godot_cpp_build_profile/build_profile.json
+          scons platform=${{ matrix.platform }} target=template_release arch=arm64 custom_api_file=thirdparty/godot_cpp_gdextension_api/extension_api.json build_profile=thirdparty/godot_cpp_build_profile/build_profile.json
+          scons platform=${{ matrix.platform }} target=template_debug arch=x86_64 custom_api_file=thirdparty/godot_cpp_gdextension_api/extension_api.json build_profile=thirdparty/godot_cpp_build_profile/build_profile.json
+          scons platform=${{ matrix.platform }} target=template_release arch=x86_64 custom_api_file=thirdparty/godot_cpp_gdextension_api/extension_api.json build_profile=thirdparty/godot_cpp_build_profile/build_profile.json
+          cd ..
+        if: matrix.platform == 'android'
```

Comment:
Since we have some AARs that contain _both_ arm64 and x86_64 (ex Khronos and MagicLeap), I think keeping them together, rather than making more items on the matrix makes the most sense.

Otherwise, we either end up with 3 new items on the matrix (like "android-arm64", "android-x86_64" and "android-arm64-plus-x86_64") or we have to make the workflow more complicated by having an earlier job that builds arm64 and x86_64 in parallel and then combines the artifacts in a subsequent job for Gradle to build the AARs.

The current way is simpler than either option.

Reply:
MagicLeap should be able to use the standard loader since "forever", fwiw. Didn't even know they had a separate one.

Reply:
We have it using the Khronos loader too. I'm not sure why we build a separate AAR file for it actually? Might be historical at this point. We can discuss on the XR team to see if we want to drop it, and just direct folks to use the Khronos AAR